crypto/crypto/src/multi_ed25519.rs

```diff
@@ -502,13 +505,21 @@ impl Signature for MultiEd25519Signature {
     ) -> Result<()> {
         // Public keys should be validated to be safe against small subgroup attacks, etc.
         precondition!(has_tag!(public_key, ValidatedPublicKeyTag));
-        let last_bit = bitmap_last_set_bit(self.bitmap);
-        if last_bit == None || last_bit.unwrap() as usize > public_key.length() {
-            return Err(anyhow!(
-                "{}",
-                CryptoMaterialError::BitVecError("Signature index is out of range".to_string())
-            ));
-        }
+        match bitmap_last_set_bit(self.bitmap) {
+            Some(last_bit) if last_bit as usize > public_key.length() => {
+                return Err(anyhow!(
+                    "{}",
+                    CryptoMaterialError::BitVecError("Signature index is out of range".to_string())
+                ))
+            }
+            Some(last_bit) => last_bit,
```

Comment:
since you don't return anything, you don't want to return `last_bit` here but `()`